composer.json

```diff
@@ -14,7 +14,8 @@
     },
     "require": {
         "php": "~7.4.4",
-        "laminas/laminas-cli": "dev-master as 0.1.0"
+        "laminas/laminas-cli": "dev-master as 0.1.0",
+        "symfony/console": "^5.0"
```

Comment:
I think we should allow also previous version of symfony/console (maybe the same as we have in laminas-cli symfony/console 4.1 || 5.0)